web/templates/web/faq.html

```diff
@@ -537,6 +538,34 @@ <h2 class="accordion-header" id="heading-20">
                 </div>
             </div>
         </div>
+        <div class="accordion-item">
+            <h2 class="accordion-header" id="heading-20-1">
+                <button class="accordion-button collapsed"
+                        type="button"
+                        data-bs-toggle="collapse"
+                        data-bs-target="#collapse-20-1"
+                        aria-expanded="false"
+                        aria-controls="collapse-20-1">
+                    {% trans "I want to review the studies I have participated in, or I have a concern and want to contact the researchers." %}
+                </button>
+            </h2>
+            <div id="collapse-20-1"
+                 class="accordion-collapse collapse"
+                 aria-labelledby="heading-20-1"
+                 data-bs-parent="#faq-accordion-participation">
+                <div class="accordion-body">
+                    <p>
+                        {% trans "Each study on Children Helping Science is run by a separate team of researchers. To find contact information for a specific study, visit your" %}
+                        <a href="{% url "web:studies-history" %}">Study History</a>
+                        {% trans "page. On this page you can also see information about when you participated in a study, and for studies that run here on our platform, watch your child's video from the session!" %}
+                    </p>
+                    <p>
+                        {% trans "If you have a more general question, want to report something to the Children Helping Science team, or need help reaching the researcher team, please contact us at" %}
+                        <a href="mailto:lookit-tech@mit.edu">lookit@mit.edu</a>!
```

Comment:
Should the `mailto` address here be the same as the text, i.e. lookit@mit.edu?

Reply:
Absolutely. I copied that a-tag from somewhere else on the page as well... so I'll have to find that too.